src/inausoft.netCLI.Tests/LogicalCommandDeserializerTests.cs

```diff
@@ -39,11 +38,56 @@ public void RegexArgumentHandler_DeserializeProperCommand_ForMultipleOptions()
             Assert.AreEqual(intOptionValue, command.IntOption);
         }
 
+        [TestMethod]
+        public void LogicalCommandDeserializer_DeserializeProperCommand_ForMultipleOptions_UsingShortNames()
+        {
+            //Arrange
+            var stringOptionValue = "sampleString";
```

Comment:
If we are using some sample values of variables i would consider making them private const at the top of the class.

Reply:
I'm fully against doing that - the less things going on outside of the test method the better - I don't see the point of moving it away - but we can have a talk.

Reply:
In three locations you use "sampleString" - string that really does not matter, so why not create private const SampleString ="sampleString" at the top and then use var stringOptionValue = SampleString?

Reply:
It is to wider discussion and can be adjusted in refactor, for now leave is as it is.